test/procedures.test.ts

```diff
@@ -188,3 +188,56 @@ test(`IN, OUT, INOUT varchar parameters`, { timeout: 15000 }, async () => {
 
   await job.close();
 });
+
+test(`IN, OUT clob parameters`, { timeout: 15000 }, async () => {
+  const job = new SQLJob();
+  await job.connect(creds);
+
+  const queryA = job.query(`
+    create or replace procedure ${TEST_SCHEMA}.procedure_test_clob(
+      in in1 clob(1m),
+      out out1 clob(1m)
+    )
+    begin
+      set out1 = upper(in1);
+    end
+  `);
+  await queryA.execute();
+  await queryA.close();
+
+  const param = "test".repeat(262144); // Create 1MB string
+  const queryB = job.query(
+    `call ${TEST_SCHEMA}.testclob(?, ?)`,
```

Comment:
```suggestion
    `call ${TEST_SCHEMA}.procedure_test_clob(?, ?)`,
```

Reply:
Sorry about that -- I had changed the procedure name and forgot to do it in both places.

Fixed.

Reply:
Thanks!